example/main.go

```diff
@@ -2,14 +2,31 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	pac.SetHelperPath(absPath("./pac-cmd"))
+	if runtime.GOOS == "darwin" {
+		pac.SetIconPathOnOSX(absPath("icon.png"))
```

Comment:
How is it unrelated to setting the pac on?  The message seems to be very closely related.  In fact, I might want the message to include the url, like "Input your password to set the system proxy to localhost:12345/pac".

Also, the way it is written, it is not thread safe and you'll need to add a mutex.